include/dts-functions.sh

```diff
@@ -322,6 +322,48 @@ board_config() {
             fi
           fi
           ;;
+          "NV54xTU")
```

Comment:
There is a slight issue in that this SMBIOS string will be the same for V540TU and V560TU. Can we add variants based on another SMBIOS string? I can set `Mainboard Version`, would that work for you?

Reply:
That would be baseboard-version in dmidecode?

Reply:
Yes, that one

Reply:
Product name is `V54x_6x_TU`, baseboard version is `V540TU` and `V560TU`

Reply:
Changed in [6c4ea5f](https://github.com/Dasharo/dts-scripts/pull/16/commits/6c4ea5f89a137abe1d7b137ceaeba066b1f9ad0e)

Reply:
Thanks

Reply:
@mkopec Actually could you set `baseboard-product-name` instead?
That way there would be less code changes as it's already displayed for example for MSI.

Reply:
Not really, unfortunately this SMBIOS string is used for Windows Update hardware identification so it should stay the same as in vendor firmware